integration-tests/integration_tests/integration_tests/http/mhs_http_request_builder.py

```diff
@@ -61,11 +61,24 @@ def execute_post_expecting_success(self) -> Response:
         """
         Execute a POST request against the MHS using the configured body and headers within this class.
         Asserts the response is successful.
-        :return: self
+        :return: response from MHS
         """
         response = requests.post(self.mhs_host, headers=self.headers, data=self.body, verify=False)
         self.assertor.assertTrue(
             response.ok,
             f'A non successful error code was returned from server: {response.status_code}')
 
         return response
+
+    def execute_post_expecting_error_response(self) -> Response:
+        """
+        Execute a POST request against the MHS using the configured body and headers within this class.
+        Asserts the response is 500.
+        :return: response from MHS
+        """
+        response = requests.post(self.mhs_host, headers=self.headers, data=self.body)
```

Comment:
This is going to need to be:
```suggestion
        response = requests.post(self.mhs_host, headers=self.headers, data=self.body, verify=False)
```
Otherwise certificate validation is going to fail. Maybe worth extracting the actual `requests.post` to a method, to avoid duplication?